### PR TITLE
Possibility to add TRD calib and fixes

### PR DESCRIPTION
**Pull request by chiarazampolli**
@catalinristea

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Pinging @martenole (I don't know Felix github account, sorry).

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 1]
```diff
@@ -186,6 +186,15 @@ if [[ $DO_TPC_RESIDUAL_EXTRACTION == "1" ]]; then
   export CALIB_CPV_GAIN=0
   export CALIB_ZDC_TDC=0
   export CALIB_FT0_TIMEOFFSET=0
+  if [[ $DO_TPC_RESIDUAL_EXTRACTION == "1" ]]; then
+    export CALIB_TPC_SCDCALIB_SENDTRKDATA=1
+    export CALIB_DIR="./"
```
Comment (martenole) [inline DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:191]: Maybe move this line up, so that its always defined in case `ADD_CALIB==1`?

**Review by martenole — APPROVED** [review]
Looks good to me

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh (hunk 1 above)
Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh]: Good idea, done!

**Comment by chiarazampolli** [comment]
Merging, we can then test this.

**Pull request merged**